app/controllers/groups_controller.rb

```diff
@@ -0,0 +1,48 @@
+class GroupsController < ApplicationController
+  before_action :set_group_params, only: %i[show edit destroy]
+
+  def index
+    @groups = Group.includes(:deals).where(user: current_user)
+    uder_deal = current_user.deals.sum(&:amount)
+    puts uder_deal
+  end
+
+  def show
+    @deals = @group.deals
+    @total_expenses = @deals.includes(:groups).where(user: current_user).sum(:amount)
+  end
```

Comment:
Hi @ekenecf. As I can see, this is the controller of the transactions page, and the deals are not ordered by the most recent.

<img width="449" alt="image" src="https://user-images.githubusercontent.com/77190203/187040780-bad2826c-2c75-43bd-80d7-47df6386b1c7.png">